bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -154,6 +188,56 @@ spec:
         displayName: Create ServiceMonitors for OpenTelemetry Collector
         path: targetAllocator.observability.metrics.enableMetrics
       version: v1alpha1
+    - description: OpenTelemetryCollector is the Schema for the opentelemetrycollectors
+        API.
+      displayName: OpenTelemetry Collector
+      kind: OpenTelemetryCollector
+      name: opentelemetrycollectors.opentelemetry.io
+      resources:
+      - kind: ConfigMaps
```

Comment:
Is this autogenerated? Aren't we missing some resources like HPA, PDB, ingress, etc?

Reply:
yes it is, the annotation is on the collector type. This is the same as for v1alpha1 and it's just a hint  for users to understand which objects are managed.

Reply:
i see. should we update it with the rest of the resources so its accurate going forward? Maybe a good separate PR...

Reply:
https://github.com/open-telemetry/opentelemetry-operator/issues/2746